store/postgres/src/relational_queries.rs

```diff
@@ -2951,6 +2959,7 @@ fn iter_column_names<'a, 'b>(
         })
         .map(|column| column.name.as_str())
         .chain(BASE_SQL_COLUMNS.iter().copied())
+        .chain(extra)
```

Comment:
You can make `extra` just a `vec!` and leave off the `.copied()` from it, as well as from `BASE_SQL_COLUMNS`

Reply:
In this case, wouldn't copying those values (`&str`) be faster than allocating memory for the new vector?

Reply:
Yes, you are right; just leave it as-is